styles/style.css

```diff
@@ -1,8 +1,172 @@
 /*
 Colors:
 
-Text: 1A1A1A
-Green: #00B172
+Text: 
+Green: 
 White: #FFF
 
 */
```

Comment:
Is this comment of any use?

Reply:
Nope, forgot to take it out, was already in there to give some hints about the coloring.
Thanks for the feedback, will implement it!